.github/GitHub.shproj

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This seems to be a misuse of the shared project. We can link just the yml file. Or add a pattern in the *.sln file to include in Common solution items. WDYT?

Reply:
I admit it's little bit of cheating, but it doesn't get build nor referenced. I haven't found a way how to achieve this in other cost-less way - I want to see entire content of `.github` directory without a need to update SLN file everytime something changes to keep maintenance cost low.

Reply:
Adding a pattern there does not help?

Reply:
I couldn't find any docs for this format in SLN file. This [SO answer](https://stackoverflow.com/a/47637436/1486185) claims it as `relative path to file = relative path to file`. I don't have a guess how to inject pattern there.
```
	ProjectSection(SolutionItems) = preProject
		..\.editorconfig = ..\.editorconfig
		src\AssemblyInfo.Shared.cs = src\AssemblyInfo.Shared.cs
		..\azure-pipelines-external.yml = ..\azure-pipelines-external.yml
		..\azure-pipelines.yml = ..\azure-pipelines.yml
		..\scripts\build\build-utils.ps1 = ..\scripts\build\build-utils.ps1
		..\scripts\build\dev-build.ps1 = ..\scripts\build\dev-build.ps1
		Directory.Build.props = Directory.Build.props
		NuGet.Config = NuGet.Config
		src\Signing.cs = src\Signing.cs
		src\SonarAnalyzer.ruleset = src\SonarAnalyzer.ruleset
		tests\SonarAnalyzer.Test.ruleset = tests\SonarAnalyzer.Test.ruleset
		StyleCopAnalyzers.targets = StyleCopAnalyzers.targets
		..\scripts\utils.ps1 = ..\scripts\utils.ps1
	EndProjectSection
```